I/IOAPI/build_tarballs.jl

```diff
@@ -24,21 +24,30 @@ mkdir $BINDIR
 sed -i 's/-ffast-math /-fPIC /g' Makeinclude* # Get rid of forbidden -ffast-math flag
 sed -i 's/-DNEED_ARGS=1/-DNEED_ARGS=1 -DIOAPI_NCF4=1/g' Makeinclude* # Specify NetCDF v4
 
-if [[ "${target}" == *i686* ]]; then
+if [[ "${nbits}" == 32 ]]; then
     sed -i 's/-m64/-m32/g' Makeinclude* # Specify 32-bit build
 elif [[ "${target}" == *aarch64* ]]; then
     sed -i 's/-m64//g' Makeinclude* # Get rid of x86 flag
 fi
 
+if [[ "${target}" == *mingw* ]]; then
+    # Add missing stdint header.
+    sed -i 's/<stdio.h>/<stdio.h>\n\#include <stdint.h>/g' bufint3.c
+
+    # This sys/wait.h header is not available and doesn't seem to be used.
+    sed -i 's/\#include <sys\/wait.h>//g' systemf.c
+fi
+
 cp Makefile.nocpl Makefile
 cp Makeinclude.Linux2_x86_64gfort Makeinclude.$BIN
 
-make fixed_src
 make # Parallel make (-j > 1) does not work
 
+cp *.h ${includedir} # C header files
 cp fixed_src/* ${includedir} # FORTRAN .EXT (include) files
 
-gfortran -shared -fPIC -fopenmp -o ${libdir}/libioapi.${dlext} -Wl,$(flagon --whole-archive) ${BINDIR}/libioapi.a -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdf -lnetcdff
+# Convert static library to dynamic library
+gfortran -shared -fPIC -fopenmp -o ${libdir}/libioapi.${dlext} -Wl,$(flagon --whole-archive) ${BINDIR}/libioapi.a -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdf -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdff
```

Comment:
```suggestion
gfortran -shared -fPIC -fopenmp -o ${libdir}/libioapi.${dlext} -L${libdir} -Wl,$(flagon --whole-archive) ${BINDIR}/libioapi.a -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdf -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdff
```

Reply:
Even with this, I get other errors:
```
sandbox:${WORKSPACE}/srcdir/ioapi-3.2/ioapi # gfortran -shared -fPIC -fopenmp -o ${libdir}/libioapi.${dlext} -L${libdir} -Wl,$(flagon --whole-archive) ${BINDIR}/libioapi.a -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdf -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdff
/workspace/srcdir/ioapi-3.2/tmp/libioapi.a(iobin3.o):iobin3.c:(.text+0x16b2): undefined reference to `fsync'
/workspace/srcdir/ioapi-3.2/tmp/libioapi.a(iobin3.o):iobin3.c:(.text+0x4040): undefined reference to `fsync'
/opt/x86_64-w64-mingw32/bin/../lib/gcc/x86_64-w64-mingw32/4.8.5/../../../../x86_64-w64-mingw32/bin/ld: /workspace/srcdir/ioapi-3.2/tmp/libioapi.a(iobin3.o): bad reloc address 0x870 in section `.rdata'
collect2: error: ld returned 1 exit status
```
`fsync` is a Unix function, I'm not sure this library is supposed to work on Windows at all: https://www.cmascenter.org/ioapi/

> System Requirements
> O/S | Linux or UNIX or MacOS

Reply:
Oh well, it was worth a try. I'd still like to add in the other changes so I will remove the windows build for now.